### PR TITLE
This PR adds Gujarati Language class along with

### PR DESCRIPTION
**Pull request by punitvara**
- stop words

## Checklist

- [x] I have submitted the spaCy Contributor Agreement.
- [x] I ran the tests, and all new and existing tests passed.
- [x] My changes don't require a change to the documentation, or if they do, I've added all required information.

### CODE REVIEW

**Comment by punitvara** [comment]
@svlandeg Can you please review and let me know if I need to do anything else ?

File: spacy/tests/lang/gu/test_text.py [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+    assert len(tokens) == 9
+
+@pytest.mark.parametrize(
+    "text,length",
+    [
+        ("ગુજરાતીઓ ખાવાના શોખીન માનવામાં આવે છે", 6),
+        ("ખેતરની ખેડ કરવામાં આવે છે.", 5),
+    ],
+)
+def test_eu_tokenizer_handles_cnts(eu_tokenizer, text, length):
```
Comment (svlandeg) [inline spacy/tests/lang/gu/test_text.py:18]: You probably want to rename this to `test_gu_tokenizer_handles_cnts` ;-)

Comment (punitvara) [reply spacy/tests/lang/gu/test_text.py:18]: Nice catch :) Thanks

Comment (svlandeg) [reply spacy/tests/lang/gu/test_text.py:18]: It also uses the `eu_tokenizer` instead of the `gu_tokenizer`...

**Comment by svlandeg** [comment]
Thanks a lot for this contribution, much appreciated !

**Pull request merged**